src/events/gps_module_event.h

```diff
@@ -33,13 +32,30 @@ enum gps_module_event_type {
 	GPS_EVT_ERROR_CODE,
 };
 
+struct gps_module_data {
+	/** GPS data timestamp. UNIX milliseconds. */
+	int64_t gps_ts;
+	/** Longitude */
+	double longi;
```

Comment:
`Longitude`? Generally, I'd avoid abbreviations in these new structs. Then comments can also be removed when the struct member names are self-explanatory